Automattic/woocommerce-payments
# Fraud & Risk tools usability improvements

Fixes #6157

#### Changes proposed in this Pull Request

Perform the usability improvements described on paJDYF-8mt-p2 that are:
- Update the review/block interface to use radio buttons
- Update the title for the metabox on a flagged and approved order
  - Remove callout link 
- Update the advanced settings CTA
  - It should be "Configure" If it's not set yet
  - It should be "Edit" when it's already set









#### Testing instructions





* The unit tests should be passing.
* Checkout to this branch
* Listen to Stripe webhooks `npm run listen`
* Start the client build `npm run start`
* Make sure the fraud settings feature flag is enabled by running `npm run wp option add wcpay_fraud_protection_settings_active 1`
* Go to **Payments > Settings**, scroll to the Fraud protection section
* Select the "Basic" protection level and save the settings
* Refresh the page
* Scroll to the Fraud protection section
* Select the "Advanced" protection level, and click on "Configure"
* Scroll to the "Order items threshold" rule, enable the screening, and set the maximum items to 3
* Set the "Filter action" to "Authorize and hold for review"
* Save the changes
* Click on the "Back to Payment Settings" button
* Scroll to the Fraud protection section
* The "Advanced" protection level CTA button should have the text "Edit" instead of "Configure"
* Go to your test site, add 4 items to the cart, and perform the checkout
* Go to **Payments > Transactions** and click on the "Risk review" tab
* Click on the order you've just created
* Click on "Approve transaction"
* Click on the order id link
* The fraud metabox should be the same as the following

<img width="284" alt="image" src="https://user-images.githubusercontent.com/16882226/236591201-1ec9043e-31cd-4b43-a995-f3e91f22f848.png">


-------------------


**Post merge**
